py-polars/polars/expr/expr.py

```diff
@@ -4609,6 +4609,18 @@ def map_elements(
               `pl.col("col_name").sqrt()`.
             - For mapping inner elements of lists, consider:
               `pl.col("col_name").list.eval(pl.element().sqrt())`.
+            - For mapping elements of struct fields, consider:
+              `pl.col("col_name").struct.field("field_name").sqrt()`.
+
+            If you want to replace the original column or field,
+            consider :meth:`.with_columns <polars.DataFrame.with_columns>`
+            and :meth:`.with_fields <polars.Expr.struct.with_fields>`.
+
+            >>> new_expr = pl.col("col_name").sqrt()
+            >>> df.with_columns(new_expr)
+
+            >>> new_expr = pl.col("col_name").struct.field("field_name").sqrt()
+            >>> df.with_columns(pl.col("col_name").struct.with_fields(new_expr))
```

Comment:
this is causing a doctest failure - I'd suggest, either remove it (you're already linking to these methods above) or use a codeblock

Reply:
@MarcoGorelli, anything else to do here?